src/calendar.ts

```diff
@@ -555,6 +556,56 @@ export class Calendar {
             }
           }
         }
+      } else if (this.options.countLockedDaysMax !== 0) {
```

Comment:
Hier möchtest du doch sicher `> 0` oder? Sonst wäre die clause bei `undefined` oder `true` auch true. Denn in der korrespondierenden If-clause oben in Zeile 514 sagen wir ja explizit nur bei `false` und der Wert hier müsste definitv ein Integer sein, wenn ich es richtig verstehe.

Reply:
Undefined sollte sie nicht sein, weil die auf den Standardwert 0 gesetzt wird. Aber dann müsste man das auch bestimmt so setzten können!